handler/src/main/java/io/netty5/handler/logging/LogLevel.java

```diff
@@ -15,32 +15,81 @@
  */
 package io.netty5.handler.logging;
 
-import io.netty5.util.internal.logging.InternalLogLevel;
+import org.slf4j.event.Level;
+
+import static java.util.Objects.requireNonNull;
 
 /**
- * Maps the regular {@link LogLevel}s with the {@link InternalLogLevel} ones.
+ * Logging level for configuring {@link LoggingHandler}.
  */
 public enum LogLevel {
```

Comment:
But shouldn't we make it package-private and hide it from the public API?

Reply:
It was the whole point to make this public and so "hide" slf4j from the public API .